android/library/src/main/java/com/liferay/mobile/screens/viewsets/defaultviews/webcontent/display/WebContentDisplayView.java

```diff
@@ -116,8 +116,24 @@ public void setScreenlet(BaseScreenlet screenlet) {
 
 	@Override
 	public boolean onTouch(View v, MotionEvent event) {
-		WebView.HitTestResult result = webView.getHitTestResult();
-		((WebContentDisplayScreenlet) getScreenlet()).onWebContentClicked(result, event);
+		boolean openUrl = ((WebContentDisplayScreenlet) getScreenlet()).onWebContentTouched(v, event);
+		boolean isClick = event.getAction() == MotionEvent.ACTION_UP;
+
+		if (!openUrl && isClick) {
+
+			WebView.HitTestResult result = webView.getHitTestResult();
+
+			//The 'event' parameter performs two actions: ACTION_DOWN and ACTION_UP.
+			LiferayLogger.i("Event: " + event.toString());
+			LiferayLogger.i("Extra: " + result.getExtra());
+
+			((WebContentDisplayScreenlet) getScreenlet()).onWebContentClicked(result);
+
+			return true;
```

Comment:
Instead of returning true its not better to return the result of the listener call?